partner_communication/models/communication_job.py

```diff
@@ -387,12 +393,14 @@ def refresh_text(self, refresh_uid=False):
         for job in self:
             lang = self.env.context.get('lang_preview', job.partner_id.lang)
             if job.email_template_id and job.object_ids:
-                fields = self.env['mail.compose.message'].with_context(
-                    lang=lang).get_generated_fields(
-                    job.email_template_id, [job.id])
+                body = job.report_id.render_qweb_html([job.id])
+                body_html = tools.html_sanitize(body[0],
+                                                sanitize_attributes=True,
+                                                sanitize_style=True,
+                                                strip_classes=True)
                 job.write({
-                    'body_html': fields['body_html'],
-                    'subject': fields['subject'],
+                    'body_html': body_html,
+                    'subject': self.subject
```

Comment:
I propose that you revert all those changes, as I never noticed such issues. If we notice problems in the future we can open another issue. But in general it's best to understand the root cause of a problem before trying something to "patch" an issue.

Reply:
ok